PubSub/src/BatchPublisher.php

```diff
@@ -77,6 +94,11 @@ public function __construct($topicName, array $options = [])
             'identifier' => sprintf(self::ID_TEMPLATE, $topicName),
             'batchMethod' => 'publishDeferred'
         ]);
+        $this->enableCompression = $options['enableCompression'] ?? false;
+        $this->compressionBytesThreshold = $options['compressionBytesThreshold'] ?? null;
```

Comment:
Docblock mentions the default as 240, but a `null` is used as default.
```suggestion
        $this->compressionBytesThreshold = $options['compressionBytesThreshold'] ?? 240;
```

Reply:
Changed.